ballerina/listener_oauth2_provider.bal

```diff
@@ -260,7 +260,7 @@ isolated function validateFromCache(cache:Cache oauth2Cache, string token) retur
         return;
     }
     if cachedEntry is any {
-        IntrospectionResponse response = <IntrospectionResponse> cachedEntry;
+        IntrospectionResponse response = <IntrospectionResponse>cachedEntry;
```

Comment:
Can't we use this:

```ballerina
isolated function cacheType(cache:Cache oauth2Cache, string token) returns oauth2:IntrospectionResponse? {
    oauth2:IntrospectionResponse|error cachedEntry = oauth2Cache.get(token).ensureType();
    if cachedEntry is oauth2:IntrospectionResponse {
        return cachedEntry;
    }
    return;
}
```

Reply:
This function will not check the exp claim to see if it is expired right?

Reply:
This seems to be a formatting change. However I have created this issue: https://github.com/ballerina-platform/ballerina-library/issues/7237 to refactor the codebase with best practices. We can handle this separately.